### PR TITLE
Backport WT-9268 series tickets

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

**Comment by quchenhao** [comment]
@kommiharibabu @lukech Please review the change commit by commit. The failures in the ppc tests are not introduced by this change. I ran a patch build on the tip of 6.0 branch and it is failing with the same issues.

https://spruce.mongodb.com/version/637c18aad6d80a54d339c0d1/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

File: src/evict/evict_page.c [hunk 1]
```diff
@@ -708,5 +708,9 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
       !__wt_page_is_modified(page) || LF_ISSET(WT_REC_HS | WT_REC_IN_MEMORY) ||
         WT_IS_METADATA(btree->dhandle));
 
+    /* Fail 0.1% of the time. */
```
Comment (kommiharibabu) [inline src/evict/evict_page.c:711]: This fail point has caused a bug and it is fixed in WT-9751. if we are going to backport this failing point, we need to backport that ticket also.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Review by lukech — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**